src/api/reactivity-utilities.md

```diff
@@ -237,7 +237,7 @@ Checks if an object is a proxy created by [`reactive()`](./reactivity-core#react
 - **Type**
 
   ```ts
-  function isProxy(value: unknown): boolean
+  function isProxy(value: any): boolean
```

Comment:
Could you please provide more context on why `any` here is preferrable to `unknown`?

Reply:
In version `3.4.22`, I have changed it to `any`.

https://github.com/vuejs/core/blob/b4b856b3f0ebe937eacb6945d13b2561037057b5/packages/reactivity/src/reactive.ts#L332

Reply:
Why was it changed in core though?

Reply:
avoid assertion

Reply:
All of the other three methods there use assertion :/

Reply:
In my impression, the refactoring has already changed the style in minor, I didn't expect it to not change yet. Let's unify them then.

https://github.com/vuejs/core/blob/bb5c31e61480bae971df6817d4a65a511bd8fa90/packages/reactivity/src/ref.ts#L42

Reply:
In my impression, the refactoring has already changed the style in minor, I didn't expect it to not change yet.

https://github.com/vuejs/core/blob/bb5c31e61480bae971df6817d4a65a511bd8fa90/packages/reactivity/src/ref.ts#L42

Reply:
I feel that avoiding assertions altogether is feasible, more concise, and the results are the same